moped-editor/src/views/staff/EditStaffView.js

```diff
@@ -93,6 +97,37 @@ const EditStaffView = () => {
                       validationSchema={validationSchema}
                       userCognitoId={userCognitoId}
                       isUserActive={isUserActive}
+                      FormButtons={({
+                        isSubmitting,
+                        watch,
+                        handleCloseModal,
+                        setModalState,
+                      }) => (
```

Comment:
Curious on thoughts about this pattern - there is form data that needs to be passed back up to these components from the `StaffForm` child for these buttons to work. I thought about moving the `useForm` hook up one level into the `NewStaffView` and `EditStaffView` components but that seemed overkill so this data gets passed as props when these function components are rendered in `StaffForm`.

Reply:
👍 i think this pattern is good!

the only thing that strikes me as a bit off is seeing the `watch("someField")` props—i want those values to flow through an "onSubmit" callback that hook-form calls: `handleSubmit(onSubmit)`. but just eyeballing things quickly, that does not seem practical—there's a lot of state going on with the modal confirmations and what not.

tldr looks good to me.

Reply:
Thanks for the feedback! That also felt off to me. I'm really hoping to roll some updates to that modal in with https://github.com/cityofaustin/atd-data-tech/issues/10085. I think it could be possible to untangle some of the modal state/props. 🤞